reana_client/cli/files.py

```diff
@@ -284,18 +301,21 @@ def upload_files(ctx, workflow, filenames, access_token):
                     sys.exit(1)
 
 
-@files_group.command(
-    'rm',
-    help='Delete the specified file or pattern.')
+@files_group.command('rm')
 @click.argument(
     'filenames',
     metavar='SOURCES',
     nargs=-1)
 @add_workflow_option
 @add_access_token_options
 @click.pass_context
-def delete_files(ctx, workflow, filenames, access_token):
-    """Delete files contained in the workflow workspace."""
+def delete_files(ctx, workflow, filenames, access_token):  # noqa: D301
+    """Delete files from workspace.
+
+    Examples:\n
+    \t $ reana-client remove -w workflow.4 data/names.txt \n
+    \t $ reana-client remove -w workflow.4 code/*
```

Comment:
I think this should be well quoted within single quotes i.e.  `-w myanalysis.42 'code/*'` otherwise the glob would be expanded by current shell already and won't reach the REANA server.

Also note data/mydata.csv as elsewhere.